ingest/ingest_files.py

```diff
@@ -82,10 +82,20 @@ class IngestFiles:
         "ignore", "Your application has authenticated using end user credentials"
     )
 
+    ALLOWED_FILE_EXTENSIONS = {
+        "text/csv": [".csv"],
+        "text/plain": [".txt"],
+        "text/tab-separated-values": [".tsv"],
+        "dataframe": [".tsv"],
+        "application/x-hdf5": [".h5ad", ".h5", ".hdf5"],
+    }
+
     def __init__(self, file_path, allowed_file_types):
         self.file_path = file_path
-        # define filetype for h5ad file extension
+        # valid suffixes for Anndata (h5ad file extensions)
```

Comment:
Blocker: this comment seems to consider the file extensions `.h5` and `.hdf5` as an `.h5ad` file extension, which seems incorrect and confusing for maintainers.

Polish: Upstream folks refer to this as "AnnData" or "anndata"; rarely or never "Anndata" in my readings.

```suggestion
        # valid suffixes for AnnData
```

Reply:
Updated comment to be more accurate `fa76fda`